features/fixtures/formatters/passed-rule.json.ts

```diff
@@ -0,0 +1,37 @@
+module.exports = [
+  {
+    description: '',
+    elements: [
```

Comment:
Given the JSON formatter’s status as soon-to-be-deprecated, and other tools’ reliance on it, would seem best to not add any additional structure for Rule/Example, and instead just fit within the roaring structure for scenarios - do we think that’s right?